javatests/build/bazel/tests/integration/WorkspaceDriverIntegrationTest.java

```diff
@@ -25,13 +25,41 @@ public void testWorkspaceWithBazelRcFile() throws Exception {
 
         driver.scratchFile(".bazelrc", "test --javacopt=\"-InvalidOpt\"");
 
-        Command cmd = driver.bazel(Optional.of(Paths.get(".bazelrc")), "test", "//:TestMe");
+        Command cmd = driver.bazelCommand()
+                .withBazelrcFile(Paths.get(".bazelrc"))
+                .withArguments("test", "//:TestMe").build();
```

Comment:
Wdyt about moving the arguments to the constructor of the builder? In the API it means moving to the “bazelCommand” method.
Two reasons:
1. It used to be mandatory to pass
2. It puts the emphasis on the args which I think are the important part (what bazel should do)

Reply:
agree 👍